requirements/dev.txt

```diff
@@ -4,7 +4,7 @@
 #
 #    pip-compile --output-file=requirements/dev.txt requirements/base.in requirements/dev.in requirements/quality.in
 #
-asgiref==3.4.1
+asgiref==3.5.0
```

Comment:
- No major change in the new version: https://github.com/aio-libs/pytest-aiohttp/blob/master/CHANGES.rst#103-2022-01-03